hail/python/hail/docs/change_log.md

```diff
@@ -1,5 +1,67 @@
 # Change Log
 
+## 0.2.13
+
+Released 2019-04-18
+
+Hail is now using Spark 2.4.x by default. If you build hail from source, you
+will need to acquire this version of Spark and update your build invocations
+accordingly.
+
+### New features
+
+- (hail#5828) Remove dependency on htsjdk for VCF INFO parsing, enabling
+  faster import of some VCFs.
+- (hail#5860) Improve performance of some column annotation pipelines.
+- (hail#5858) Add `unify` option to `Table.union` which adds special behavior
+  when duplicate rows are present.
+- (hail#5756) Hail now uses Spark 2.4.x by default.
+- (hail#5677) `MatrixTable` now also supports `show`.
+- (hail#5790) Add `array.head()` which returns the first element of the array,
+  or missing if the array is empty.
+- (hail#5690) Add `_localize` flag to `locus_windows` to prevent serialization
+  in `ld_matrix`.
+- (hail#5743) `mt.compute_entry_filter_stats` computes statistics about the number
+  of filtered entries in a matrix table.
+- (hail#5758) failure to parse an interval will now produce a much more detailed
+  error message.
+- (hail#5723) `hl.import_matrix_table` can now import a matrix table with no
+  columns.
+- (hail#5724) `hl.rand_norm2d` samples from a two dimensional random normal.
+
+### Bug fixes
+
+- (hail#5885) Fix `Table.to_spark` in the presence of fields of tuples.
+- (hail#5882)(hail#5886) Fix `BlockMatrix` conversion methods to correctly
+  handle filtered entries.
+- (hail#5884)(hail#4874) Fix longstanding crash when reading Hail data files
+  under certain conditions.
+- (hail#5855) Fix `hl.mendel_errors` incorrectly reporting children counts in
+  the presence of entry filtering.
+- (hail#5830)(hail#5835) Fix Nirvana support
+- (hail#5853)(hail#5831) Fix `mt.entries()` error when the matrix table has no
```

Comment:
This bug didn't exist in the last release.

The entries() 4x performance improvement should be here though!